default/scripting/empire_statistics/Empire.focs.txt

```diff
@@ -24,3 +24,11 @@ Statistic name = "PLANETS_DEPOPULATED" value =
 
 Statistic name = "PLANETS_INVADED" value =
     SpeciesPlanetsInvaded empire = Source.Owner
+
+Statistic name = "TOTAL_POPULATION_STAT"
+    value = Statistic Sum
+        value = Source.Population
```

Comment:
I doubt you really mean / want `Source.Population`. Probably `LocalCandidate.Population`.

See: https://github.com/freeorion/freeorion/blob/master/universe/ValueRef.h#L903